### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
looks prefect to me

File: Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+/* text-avatar-styling */
+.text{
+    background-color: var(--color-bg);
+    text-align: center;
+    vertical-align: center;
+    font-size: smaller;
+    color: black;
+    font-weight: bolder;
```
Comment (anupkgurung) [inline Avatar/avatar.css:48]: add color to root and used variables.

File: Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+@import url(../styles.css);
+
+:root{
+    --color-bg:rgb(238, 157, 5);
+}
```
Comment (anupkgurung) [inline Avatar/avatar.css:5]: color name would be meaningful eg: --white-bg

**Review by anupkgurung — APPROVED** [review]
looks good !!
Have added comment

File: Avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="avatar.css">
+    <script src="https://kit.fontawesome.com/60d2206e29.js" crossorigin="anonymous"></script>
+    </head>
+<body>
+
+<div class="container">
+    <!-- circle avatar ------------------------------->
```
Comment (smilegupta) [inline Avatar/avatar.html:14]: Comment is wrong..

File: Avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          <span class="avatar-text ">FOCUS</span>
+    </div>
+</div>
+
+
+
+
+
+
+
```
Comment (smilegupta) [inline Avatar/avatar.html:65]: Why leaving empty spaces

File: Avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+</div>
+</body>
+</html>
```
Comment (smilegupta) [inline Avatar/avatar.html:68]: Code Indentation is missing

File: Avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="avatar.css">
+    <script src="https://kit.fontawesome.com/60d2206e29.js" crossorigin="anonymous"></script>
```
Comment (smilegupta) [inline Avatar/avatar.html:9]: Include sctipt tag either in bottom or use defer

**Review by smilegupta — CHANGES_REQUESTED** [review]
Added a few comments, have a look please

File: Avatar/avatar.html (hunk 6 above)
Comment (ishant005) [reply Avatar/avatar.html:9]: will fix it

**Pull request merged**